docs/sortinghat/profiles/add.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+title: Creating profiles
+nav_order: 1
+parent: Profiles and Identities
+grand_parent: SortingHat
+has_children: false
+has_toc: false
+---
+
+# How to create a profile
+
+Creating a profile is akin to adding an individual in SortingHat.
+
+To add an individual, click on the **Add** button in the `Individuals` table.
+This will open a dialogue titled `Add individual`. Fill in the individual's
+respective information and finally click on **Save** to save the information.
+
+![create-profile](../assets/create-profile.png)
+
+In the **Add individual** dialogue,
+
+- Name: `Name of the individual`
```

Comment:
I think it would be better if we can change the formatting.

- `Name`: Name of the individual

WDYT?

Reply:
Yeah. Its usually like that normally. Ok I'll change it